soh/soh/SaveManager.cpp

```diff
@@ -989,6 +998,14 @@ void SaveManager::LoadBaseVersion2() {
         SaveManager::Instance->LoadArray("dungeonKeys", ARRAY_COUNT(gSaveContext.sohStats.dungeonKeys), [](size_t i) {
             SaveManager::Instance->LoadData("", gSaveContext.sohStats.dungeonKeys[i]);
         });
+        SaveManager::Instance->LoadData("playTimer", gSaveContext.sohStats.playTimer);
+        SaveManager::Instance->LoadData("pauseTimer", gSaveContext.sohStats.pauseTimer);
+        SaveManager::Instance->LoadArray("timestamps", ARRAY_COUNT(gSaveContext.sohStats.timestamp), [](size_t i) {
+            SaveManager::Instance->LoadData("", gSaveContext.sohStats.timestamp[i]);
+        });
+        SaveManager::Instance->LoadArray("counts", ARRAY_COUNT(gSaveContext.sohStats.count), [](size_t i) {
+            SaveManager::Instance->LoadData("", gSaveContext.sohStats.count[i]);
+        });
```

Comment:
will this cause any issues with loading old saves? i would assume it's fine considering these are new as opposed to modified, but it'd be nice to confirm

Reply:
I will try setting up some save files on 5.0.0 and see what happens loading them into this.

Reply:
Loading an older save doesn't seem to create any obvious problems! Obviously the timers and stats will be off for that particular save file but that's to be expected. @briaguya-ai